### ISSUE TITLE
Package.activateConfig is deprecated.

### ISSUE THREAD

**Issue by FernandoZueet** [AtomLinter/linter-php#23]
Use a config schema instead. See the configuration section
of https://atom.io/docs/latest/hacking-atom-package-word-count and
https://atom.io/docs/api/latest/Config for more details
```
Package.activateConfig (C:\Users\Fernando\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:267:11)
Package.activateNow (C:\Users\Fernando\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:238:14)
<unknown> (C:\Users\Fernando\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:223:30)
Package.measure (C:\Users\Fernando\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:167:15)
```

**Comment by joshbadger** [comment]
I'm experiencing the exact same issue.

**Comment by kevinsawicki** [comment]
Fixed by #29  and published in `0.0.13`
